src/easyscience/Objects/Job/Job.py

```diff
@@ -74,10 +74,10 @@ def analysis(self, analysis: AnalysisBase):
     # def info(self, info: InfoBase):
     #     raise NotImplementedError("info setter not implemented")
 
-    @abstractmethod
-    def calculate_theory(self, *args, **kwargs):
-        raise NotImplementedError("calculate_theory not implemented")
-        #pass
+    # @abstractmethod
```

Comment:
Should this be deleted?

Reply:
The summary/info abstract classes will be added later, so I left the code for future reuse.
The `calculate_theory` abstract method should not be commented out. Fixed.